src/helm.ts

```diff
@@ -46,6 +46,24 @@ export const installUrbanOS = (
     timeout: 600,
   });
 
+export const installSauron = (
+  classRef: TerraformStack,
+  dependsOn: DependsOn
+) =>
+  new Release(classRef, "SauronHelmRelease", {
+    name: "sauron",
+    chart: "sauron",
+    version: "0.0.17",
```

Comment:
here has Sauron at 0.0.17, whereas sauron_values.yaml has 0.0.8. Just making sure all is good there and hopefully consistent

Reply:
Good catch 👍